src/state/slices/instructor/actions/index.js

```diff
@@ -1,37 +1,37 @@
-import * as InstructorScreenActionType from './InstructorScreenActionType';
+import * as instructorActionTypes from './instructorActionTypes'
```

Comment:
What advantage is gained by moving these up an out of Instructors module? Is this related to the GetPublished module being treated as a sibling of Instructors?